### PR TITLE
docs: fixes for some issues I found

### PR DESCRIPTION
**Pull request by fabis94**
When going through the docs noticed some issues and inaccuracies. These are the ones I was able to fix on my own.

### CODE REVIEW

File: dev/server-rest-api.md [hunk 1]
```diff
@@ -17,7 +17,9 @@ This method is useful for optimizing uploads, so that objects that are already i
 
 **Required permissions:** Write permissions on the stream
 
-**Parameters:**
```
Comment (fabis94) [inline dev/server-rest-api.md:20]: This page wasn't clear on the request body format. There are many possibilities - sending JSON in the request body, sending form-data, using x-www-form-urlencoded etc.

File: user/FAQs.md [hunk 2]
```diff
@@ -13,7 +13,7 @@ Speckle is a swiss-army knife for working with 3D data and we keep adding new fu
 - 3D Viewer: see your CAD and BIM models online, share and embed them anywhere
 - Real time: get real time updates and notifications and changes
 - Object-based: say goodbye to files! Speckle is the first object based platform for the AEC industry, store your data in a fast and accessible database
-- Version control: Speckle is the Git & Hub for geometry and BIM data
+- Version control: Speckle is the Git & Hub for geometry and BIM data - you can manage multiple versions of this data with ease!
```
Comment (fabis94) [inline user/FAQs.md:16]: The Version control is a very foundational feature for Speckle, but it's explained here in a pretty unclear way. Non-devs don't know what GitHub or "Git & Hub" means, so it's important to specify what this means

File: user/FAQs.md [hunk 3]
```diff
@@ -79,7 +79,7 @@ Currently, commits, branches or streams cannot be merged via Speckle, but you ca
 
 ## Why can't I see anything in the 3D Viewer?
 
-Our 3D Viewer only supports visualizing geometry. If you're sending any data that doesn't have a supported geometrical representation it will not be visible, for example when sending a list of Levels or Revit Family and Type names. You'll always be able to explore the raw data of a Stream in the JSON editor inside the commit page.
+Our 3D Viewer only supports visualizing geometry. If you're sending any data that doesn't have a supported geometrical representation it will not be visible, for example when sending a list of Levels or Revit Family and Type names. You'll always be able to explore the raw data of a Stream in the commit page, however.
```
Comment (fabis94) [inline user/FAQs.md:82]: Original text was misleading, there is no JSON editor there

File: user/web.md [hunk 4]
```diff
@@ -9,7 +9,6 @@ The **Speckle Web App** is our browser-based interface for managing all things S
 - Creating and editing streams
 - Viewing your data in our 3D model viewer
 - Managing your account
-- Sharing your data with others
```
Comment (fabis94) [inline user/web.md:12]: Redundant, "Coordinating your data among various parties & users" already covers this

File: user/web.md [hunk 5]
```diff
@@ -114,22 +113,19 @@ Your stream must be made **public** in order for the embedded viewer to properly
 
    ![Stream page with geometric data](./img/web/embedViewer-StreamPage.png)
 
-2. Press the _play_ button to load the viewer. The viewer toolbar should appear in the bottom.
```
Comment (fabis94) [inline user/web.md:117]: This section was outdated, it seems. I don't see any sharing/embed options in the toolbar, these options are now available through the "Share" modal

Comment (fabis94) [reply user/web.md:117]: Also it's unnecessary to increment the leading numbers of a numbered list in markdown, each list item can start with "1." and when the MD file is rendered the list numbers will be rendered correctly

**Comment by didimitrie** [comment]
@fabis94, this looks good - feel free to merge away!

**Pull request merged**